modules/nf-core/semibin/singleeasybin/main.nf

```diff
@@ -0,0 +1,45 @@
+process SEMIBIN_SINGLEEASYBIN {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::semibin=1.4.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/semibin%3A1.4.0--pyh7cba7a3_0':
+        'quay.io/biocontainers/semibin%3A1.4.0--pyh7cba7a3_0' }"
```

Comment:
```suggestion
        'https://depot.galaxyproject.org/singularity/semibin:1.4.0--pyh7cba7a3_0':
        'quay.io/biocontainers/semibin:1.4.0--pyh7cba7a3_0' }"
```

Maybe this will work?

Reply:
Thanks! 

For the conda, if there is a way to add another channel (conda-forge) here: conda "bioconda::semibin=1.4.0"?

Sincerely
Shaojun

Reply:
I think this maybe can work.

Reply:
The problem with the conda should have been fixed now (by merging with the `master` branch) It was a known problem :)